### PR TITLE
 adding gift wish

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/screens/Gift/styles.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.gift {
+  display: flex;
+  flex-direction: row;
+  height: 50px;
+  padding: 6px 10px;
+  background-color: red;
```
Comment (mi-chal) [inline src/screens/Gift/styles.scss]: Colors should be kept in separate file.

File: src/screens/Gift/styles.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  margin-top: 5px;
+}
+
+.gift__input-name {
+  border: none;
+  padding: 6px 10px;
+  border-radius: 5px;
+  font-size: 1.2rem;
+  width: 90%;
+  font-family: monospace, Arial, Helvetica, sans-serif;
```
Comment (mi-chal) [inline src/screens/Gift/styles.scss]: Same with fonts

File: src/shared/components/PersonCard/styles.scss [hunk 3]
```diff
@@ -95,4 +59,13 @@
...
+}
+
+.person-card__gifts-list {
+  display: flex;
+  flex-direction: column;
+}
+
+
+
+
```
Comment (mi-chal) [inline src/shared/components/PersonCard/styles.scss]: Remove

File: src/store/gifts/reducer.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import generateId from '../../utils/idGenerator';
```
Comment (mi-chal) [inline src/store/gifts/reducer.js]: Use absolute import

**Review by mi-chal — APPROVED** [review]

File: src/shared/components/PersonCard/styles.scss (hunk 3 above)
Comment (dwantula) [reply src/shared/components/PersonCard/styles.scss]: Done!

File: src/store/gifts/reducer.js (hunk 4 above)
Comment (dwantula) [reply src/store/gifts/reducer.js]: Fixed

File: src/screens/Gift/styles.scss (hunk 1 above)
Comment (dwantula) [reply src/screens/Gift/styles.scss]: Done!

File: src/screens/Gift/styles.scss (hunk 2 above)
Comment (dwantula) [reply src/screens/Gift/styles.scss]: Done!

**Pull request merged**